packages/core/package.json

```diff
@@ -523,7 +523,7 @@
         "aws-ssm-document-language-service": "^1.0.0",
         "bytes": "^3.1.2",
         "cross-fetch": "^4.0.0",
-        "cross-spawn": "^7.0.3",
```

Comment:
Did you mean to merge this change into here? Do we want this going to master?

Reply:
thanks for the check; this was temporary, since i was trying to get codecov to actually run. but it turns out the problem is that it never runs in GHA...